path_login.go

```diff
@@ -436,18 +439,35 @@ func (b *azureAuthBackend) verifyResource(ctx context.Context, subscriptionID, r
 			}
 		}
 
+		// add the identities associated with the bound resource groups, if any
+		for _, rg := range role.BoundResourceGroups {
+			pager := c.NewListByResourceGroupPager(rg, &armmsi.UserAssignedIdentitiesClientListByResourceGroupOptions{})
```

Comment:
Nit: could we either refactor this common code into a helper function or append `resourceGroupName` to this array?

Reply:
yeah i think appending it would work